release/release-github

```diff
@@ -68,7 +68,7 @@ message()
 json_escape()
 {
     # Escape quotes, backslashes and new lines
-    sed -e 's/[\\"]/\\\0/g' -e 's/$/\\n/g'
+    sed -e 's/[\\"]/\\\0/g' -e 's/$/\\n/g' | tr -cd '[\t -~]'
```

Comment:
This will remove the line breaks too, was that intended? Before:
```
$ git for-each-ref --format='%(contents)' "refs/tags/135" | sed -e '/^\s*135\s*$/d' -e '/-----BEGIN/,/-----END/d' | sed -e 's/[\\"]/\\\0/g' -e 's/$/\\n/g'
 * It is now possible to use file descriptors passed over the DBus API\n
 * Add \"Disks\" tab to Virtual Machines\n
 * Hide the top navigation bar if empty, i.e. if dashboard is not available\n
 * Redesign the page menu and machine switcher\n
 * Show \"Locked/Unlocked\" indicator for privilege escalation in the top bar\n
 * Make privilege escalation work with sudo reauthorization\n
 * Add developer tool for closing active pages\n
 * Use official /etc/ssh/ssh_known_hosts for adding remotely managed machines\n
 * Connect to other machines with SSH via cockpit-bridge instead of cockpit-ws\n
\n
```

Now:
```
$ git for-each-ref --format='%(contents)' "refs/tags/135" | sed -e '/^\s*135\s*$/d' -e '/-----BEGIN/,/-----END/d' | sed -e 's/[\\"]/\\\0/g' -e 's/$/\\n/g' | tr -cd '[\t -~]'
 * It is now possible to use file descriptors passed over the DBus API\n * Add \"Disks\" tab to Virtual Machines\n * Hide the top navigation bar if empty, i.e. if dashboard is not available\n * Redesign the page menu and machine switcher\n * Show \"Locked/Unlocked\" indicator for privilege escalation in the top bar\n * Make privilege escalation work with sudo reauthorization\n * Add developer tool for closing active pages\n * Use official /etc/ssh/ssh_known_hosts for adding remotely managed machines\n * Connect to other machines with SSH via cockpit-bridge instead of cockpit-ws\n\n
```

Also using a character class "space to tilde", while clever, looks a bit dubious, and it's prone to miss legitimate printable characters including non-English letters/symbols. How about a more explicit
```
$ git for-each-ref --format='%(contents)' "refs/tags/135" | sed -e '/^\s*135\s*$/d' -e '/-----BEGIN/,/-----END/d' | sed -e 's/[\\"]/\\\0/g' -e 's/$/\\n/g' | tr -cd '[[:print:][:space:]]'
 * It is now possible to use file descriptors passed over the DBus API\n
 * Add \"Disks\" tab to Virtual Machines\n
 * Hide the top navigation bar if empty, i.e. if dashboard is not available\n
 * Redesign the page menu and machine switcher\n
 * Show \"Locked/Unlocked\" indicator for privilege escalation in the top bar\n
 * Make privilege escalation work with sudo reauthorization\n
 * Add developer tool for closing active pages\n
 * Use official /etc/ssh/ssh_known_hosts for adding remotely managed machines\n
 * Connect to other machines with SSH via cockpit-bridge instead of cockpit-ws\n
\n
```